specs/laravel/laravel_create_new_project.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: Create a new Laravel App using Composer
+command: composer create-project laravel/laravel example-app
+tags:
+  - composer
+  - laravel
+  - php
+description: ~
```

Comment:
Are there any relevant descriptions here that would make it clearer what this command does?

Reply:
Yeap no worries, just updated the command descriptions

Reply:
Thanks for the review btw <3